# This file is part of the RonDB REST API Server
# Copyright (c) 2025 Hopsworks AB
#
# This program is free software: you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation, version 3.
#
# This program is distributed in the hope that it will be useful, but
# WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
# General Public License for more details.
#
# You should have received a copy of the GNU General Public License
# along with this program. If not, see <http://www.gnu.org/licenses/>.

Symlink trantor to where drogon expects to find it as a submodule.

diff --git a/storage/ndb/rest-server2/extra/drogon/drogon-1.9.7/trantor b/storage/ndb/rest-server2/extra/drogon/drogon-1.9.7/trantor
new file mode 120000
index 00000000000..c7224976042
--- /dev/null
+++ b/storage/ndb/rest-server2/extra/drogon/drogon-1.9.7/trantor
@@ -0,0 +1 @@
+../trantor-1.5.21/
\ No newline at end of file
